tests/TableExportsTests.cs

```diff
@@ -37,6 +37,33 @@ public void ItHasTheExpectedNumberOfExportedTables()
             GetTableExports().Count().Should().Be(Fixture.Module.Exports.Tables.Count);
         }
 
+        [Fact]
+        public void ItCreatesExternsForTheGlobals()
```

Comment:
Should this be `ItCreatesExternsForTheTables`?

Reply:
Fixed it, should be good now